### ISSUE TITLE
I have warnings in the error file that I do not understand.

### ISSUE THREAD

**Issue by ewars** [simroux/VirSorter#90]
I have the following errors in the "err" file:

[mclIO] writing </home/earaya/resultados/r_3/new_clusters.mci>
.......................................
[mclIO] wrote native interchange 10469x10469 matrix with 10936 entries to stream </home/earaya/resultados/r_3/new_clusters.mci>
[mclIO] wrote 10469 tab entries to stream </home/earaya/resultados/r_3/new_clusters.tab>
[mcxload] tab has 10469 entries
[mclIO] reading </home/earaya/resultados/r_3/new_clusters.mci>
.......................................
[mclIO] read native interchange 10469x10469 matrix with 10936 entries
[mcl] pid 115899
 ite   chaos  time hom(avg,lo,hi) m-ie m-ex i-ex fmv
  1     0.51  0.02 1.00/0.80/1.17 1.01 1.01 1.01   0
  2     0.51  0.01 1.00/0.75/1.01 1.00 1.00 1.01   0
  3     0.62  0.01 1.00/0.69/1.00 1.00 0.99 1.01   0
  4     0.70  0.02 1.00/0.71/1.00 1.00 0.99 0.99   0
  5     0.67  0.02 1.00/0.72/1.00 1.00 0.99 0.99   0
  6     0.69  0.01 1.00/0.73/1.00 1.00 1.00 0.98   0
  7     0.75  0.02 1.00/0.68/1.00 1.00 1.00 0.98   0
  8     0.62  0.01 1.00/0.72/1.00 1.00 1.00 0.98   0
  9     0.37  0.03 1.00/0.76/1.00 1.00 1.00 0.98   0
 10     0.25  0.02 1.00/0.76/1.00 1.00 1.00 0.98   0
 11     0.25  0.01 1.00/0.76/1.00 1.00 1.00 0.97   0
 12     0.25  0.01 1.00/0.76/1.00 1.00 1.00 0.97   0
 13     0.25  0.02 1.00/0.76/1.00 1.00 1.00 0.97   0
 14     0.25  0.01 1.00/0.76/1.00 1.00 1.00 0.97   0
 15     0.24  0.01 1.00/0.77/1.00 1.00 1.00 0.97   0
 16     0.24  0.01 1.00/0.76/1.00 1.00 1.00 0.96   0
 17     0.23  0.01 1.00/0.77/1.00 1.00 1.00 0.96   0
 18     0.21  0.02 1.00/0.83/1.00 1.00 1.00 0.96   0
 19     0.25  0.01 1.00/0.76/1.00 1.00 1.00 0.96   0
 20     0.17  0.01 1.00/0.83/1.00 1.00 1.00 0.96   0
 21     0.03  0.01 1.00/0.97/1.00 1.00 1.00 0.96   0
 22     0.00  0.02 1.00/1.00/1.00 1.00 1.00 0.96   0
 23     0.00  0.01 1.00/1.00/1.00 1.00 1.00 0.96   0
[mcl] jury pruning marks: <100,100,99>, out of 100
[mcl] jury pruning synopsis: <99.9 or perfect> (cf -scheme, -do log)
[mcl] output is in /home/earaya/resultados/r_3/new_clusters.csv
[mcl] 5234 clusters found
[mcl] output is in /home/earaya/resultados/r_3/new_clusters.csv

Please cite:
    Stijn van Dongen, Graph Clustering by Flow Simulation.  PhD thesis,
    University of Utrecht, May 2000.
       (  http://www.library.uu.nl/digiarchief/dip/diss/1895620/full.pdf
       or  http://micans.org/mcl/lit/svdthesis.pdf.gz)
OR
    Stijn van Dongen, A cluster algorithm for graphs. Technical
    Report INS-R0010, National Research Institute for Mathematics
    and Computer Science in the Netherlands, Amsterdam, May 2000.
       (  http://www.cwi.nl/ftp/CWIreports/INS/INS-R0010.ps.Z
       or  http://micans.org/mcl/lit/INS-R0010.ps.Z)

BLAST options error: File /home/earaya/resultados/r_3/db/Pool_new_unclustered.faa is empty
BLAST Database error: No alias or index file found for protein database [/home/earaya/resultados/r_3/db/Pool_new_unclustered] in search path [/home/earaya::]

**Comment by simroux** [comment]
Right, this does not look like a problem. MCL (the clustering process) gives its output in the "standard error" file, but that is no cause for concern. Similarly, the "BLAST options error: File /home/earaya/resultados/r_3/db/Pool_new_unclustered.faa is empty" line and the next ones are warnings but not real errors (it is not uncommon to have this file be empty, especially on round 3). So everything looks ok, and if you got all expected results files then there is no reason to worry ! 

Best,
Simon

**Comment by ewars** [comment]
Excellent. Thank you very much for your reply.
